inc/config.h

```diff
@@ -189,9 +133,11 @@
 #define SOFTWARE_SERIAL_BAUD 9600
 
 // ############################### SERIAL PROTOCOL ###############################
-// enables processing of input characters through 'protocol.c'
-#define INCLUDE_PROTOCOL
+#define NO_PROTOCOL 0
```

Comment:
See [here](https://github.com/btsimonh/hoverboard-firmware-hack/pull/6/files#diff-#diff-8d079de56f4a44c5abf757fcf13777baR223])

Reply:
@EmerickH @p-h-a-i-l  - we are the only people (currently) maybe using the 'old' protocol.  So I'm not sure we need to retain it in the long term.  Reviewing the discussions, we didn't find anything specifically 'bad' with the proposed protocol.
One of the issues here is that the new protocol has the CI near the start, so the processing (and and sending code) would need to be different unless we 'shim' it (which would involve more copying/buffers, etc).  So I'm thinking we just move to the new, rework the processing and all sends to know the new structure (and make things like 1+1+1+len+1 a little nicer :) ).
One thing we should do is TX queuing.  At the moment only the last TX message is stored (so we can resend), but the first thing that will happen is we'll do @p-h-a-i-l  's regular sends, and we'll end up trying to send two things at the same time, and while we have buffer space, I think both should be sent....
I'm still up to my eyes in 'real' work, but since the HB is currently out and fully charged, I'll try to get some time on this today :).  just complete the implementation and go for new protocol testing.....